### ISSUE TITLE
Allow spinnaker to set securityContext of a Pod in Kubernetes

### ISSUE THREAD

**Issue by dennismartensson** [spinnaker/spinnaker#1284]
*Title*

Allow spinnaker to set securityContext of a Pod in Kubernetes

*Cloud Provider*

Kubernetes

*Environment*

I am running Spinnaker on GCP.

*Feature Area*

Allow spinnaker to set the securityContext of a Pod so it would need to backend stuff in clouddriver and some UI to go with it. 

*Description*

Would like to be able to run pods with higher privileged to for example use 
[gcsfuse](https://github.com/GoogleCloudPlatform/gcsfuse). This would need to be deployed to my Kubernetes pods and configured in the UI.

```
securityContext:
     capabilities: {}
     privileged: true 
```

**Comment by lwander** [comment]
So I added support for container level security contexts because those are called "securityContext", and have "privileged" & "capabilities" fields; however, these aren't pod level "podSecurityContexts". I can add those next if this is a misunderstanding, but for now, you can find the PR for backend support here: https://github.com/spinnaker/clouddriver/pull/1229

**Comment by dennismartensson** [comment]
I think this looks good! Thank you @lwander

**Comment by lwander** [comment]
Passing this on to you, @danielpeach

**Comment by dennismartensson** [comment]
@danielpeach Would be very nice if you could get this in place we really like to use it.

Thank you for the fast work @lwander

**Comment by lwander** [comment]
Closed in Deck 2.919.0 and Clouddriver 1.438.0

**Comment by dennismartensson** [comment]
Have done some test deploys and working very good so far.
